dt-posts/dt-posts.php

```diff
@@ -1423,6 +1423,52 @@ public static function get_post_tiles( $post_type, $return_cache = true ){
         return $tile_options[$post_type];
     }
 
+    /**
+     * Request record access
+     *
+     * @param string $post_type
+     * @param int $post_id
+     * @param int $user_id -- Requesting user
+     *
+     * @return false|int|WP_Error
+     */
+    public static function request_record_access( string $post_type, int $post_id, int $user_id ) {
```

Comment:
It will be more secure to use `get_current_user_id()` instead of passing in the $user_id.